src/main/proto/ga4gh/common.proto

```diff
@@ -5,6 +5,16 @@ syntax = "proto3";
 
 package ga4gh;
 
+// When returning an HTTP error response, a server may also return a JSON formatted GAException
+// to better describe the error.
+message GAException {
+  // Numerical error code (should match the HTTP response code).
```

Comment:
This is a new requirement, and contrary to how the reference server uses the status code. We should revert to using the old comments and update this later if necessary.